FakeItEasy.sln

```diff
@@ -1,26 +1,32 @@
-﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0

```

Comment:
OK, fair enough. The fact that it opens for you in VS2015 now instead of VS2013 is indeed a change. I'll revert these lines. Would you prefer a commit amend and force push? Or a new commit?

Incidentally, I prefer the fact that it opens in VS2015 by default with this change. It continually annoyed me when it opened in VS2013 by default. But anyway, we can discuss the VS versioning aspect elsewhere - it shouldn't hold up this PR.